app/pages/reporter/complete-submit.tsx

```diff
@@ -16,34 +18,84 @@ class CompleteSubmit extends Component<Props> {
-    query completeSubmitQuery {
+    query completeSubmitQuery($applicationId: ID!) {
       query {
         session {
           ...defaultLayout_session
         }
+        application(id: $applicationId) {
+          facilityByFacilityId {
+            facilityName
+            organisationByOrganisationId {
+              operatorName
+            }
+          }
+          latestSubmittedRevision {
+            versionNumber
+          }
+        }
       }
     }
   `;
 
   render() {
     const {
-      query: {session}
+      query: {session, application},
+      router
     } = this.props;
+
+    const {applicationId} = router.query;
+    const {versionNumber} = application.latestSubmittedRevision;
+    const {facilityName} = application.facilityByFacilityId;
+    const {
+      operatorName
+    } = application.facilityByFacilityId.organisationByOrganisationId;
+
     return (
-      <DefaultLayout
-        session={session}
-        title={
-          <>
-            Thank you for your submission. Your application has been sent and is
-            being reviewed.
-            <br /> We will notify you as soon as there is an update.
-          </>
-        }
-      >
+      <DefaultLayout session={session}>
+        <Row className="justify-content-md-center mb-5">
+          <Col>
+            <ProgressStepIndicator
+              title="Your Facility's Application Status"
+              steps={[
+                {
+                  description: 'Complete Facility CIIP Application',
+                  badgeStyle: StatusBadgeColor.NONE,
+                  number: 1,
+                  completed: true
+                },
+                {
+                  description: 'Submit Completed Application',
+                  badgeStyle: StatusBadgeColor.INITIAL,
+                  number: 2,
+                  completed: true
+                },
+                {
+                  description: 'Application Reviewed by Administrators',
+                  badgeStyle: StatusBadgeColor.PENDING,
+                  number: 3
+                },
+                {
+                  description: 'Application Approved or Rejected',
+                  badgeStyle: StatusBadgeColor.APPROVED,
+                  number: 4
+                }
+              ]}
+            />
+          </Col>
+        </Row>
+        <h3 className="pt-5">Thank you for submitting your application.</h3>
+        <p className="lead pb-5 pt-3">
+          Your application for the{' '}
+          <span className="font-weight-bold">{facilityName}</span> facility
+          operated by <span className="font-weight-bold">{operatorName}</span>{' '}
+          has been received and is being reviewed. We will notify you by email
+          regarding any updates.
+        </p>
         <Link
           passHref
           href={{
-            pathname: '/reporter'
+            pathname: `/reporter/application/${applicationId}/version/${versionNumber}/view`
```

Comment:
There's a function in `routes.ts` that you can import to get the route. There's a TechDebt card that was started to make routes usage more DRY